src/buildercore/terraform.py

```diff
@@ -401,27 +398,38 @@ def render_bigquery(context):
         } for dataset_id, options in context['bigquery'].items()
     }
 
-    if tables:
+    def add_table(table_id, table_options):
+        schema = table_options['schema']
+        stackname = context['stackname']
+        fqrn = "%s_%s" % (table_options['dataset_id'], table_id) # 'fully qualified resource name'
```

Comment:
it was not any standard terminogy, could just be called `unique_name` while inside this scope